src/pages/AppLogin.vue

```diff
@@ -38,7 +41,8 @@
 }
 
 .ripple:hover {
```

Comment:
these `.ripple` styles are already in `index.css` in root directory, I guess we can use that?